fhir-persistence-jdbc/src/main/java/com/ibm/fhir/persistence/jdbc/util/QuerySegmentAggregator.java

```diff
@@ -43,7 +43,7 @@
     private static final String FROM = " FROM ";
     private static final String UNION = " UNION ALL ";
     protected static final String ON = " ON ";
-    private static final String AND = " AND ";
+    private static final String JOIN = " JOIN ";
```

Comment:
I think we should push these constants into JDBCConstants and statically import. 

Is AND no longer used?

Reply:
yeah, not reference to ADN anymore

Reply:
ugly, just noticed that there were some constants in JDBCConstants, and some are defined in QuerySegmentAgg. and there are overlaps, I'm going to refactor all of them